libs/form/src/next/upload/stories/vl-upload.stories-arg.ts

```diff
@@ -0,0 +1,192 @@
+import { CATEGORIES, TYPES } from '@domg-wc/common-storybook';
+import { ArgTypes } from '@storybook/web-components';
+import { uploadDefaults } from '../vl-upload.component';
+import { formControlArgs, formControlArgTypes } from '../../form-control/stories/form-control.stories-arg';
+import { action } from '@storybook/addon-actions';
+
+type UploadArgs = typeof uploadDefaults &
+    typeof formControlArgs & {
+        onVlInput: () => void;
+        onVlError: () => void;
+        onVlValid: () => void;
+    };
+
+export const uploadArgs: UploadArgs = {
+    ...formControlArgs,
+    ...uploadDefaults,
+    onVlInput: action('vl-input'),
+    onVlValid: action('vl-valid'),
+    onVlError: action('vl-error'),
+};
+
+export const uploadArgTypes: ArgTypes<typeof uploadArgs> = {
```

Comment:
Hier kan je het `UploadArgs ` type hergebruiken, zie net dat ik dat ook niet gedaan heb bij de andere componenten, zal dit meenemen in mijn verbeter ticket.

Reply:
Makes sense. Will do